utils/configure-project

```diff
@@ -35,10 +36,12 @@ configure_circleci_project() {
 
 usage() {
     cat <<EOF
-usage: $0 OPTIONS <service-name>
+usage: $(basename "$0") OPTIONS <service-name>
```

Comment:
Should be cuter, like this. :heart:

Reply:
No, because it hides an important information (the fact that the script is not in the PATH and is called from a specific place).